File: PharmacyApp/src/main/java/com/pharmacy/model/Objection.java
package com.pharmacy.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "objections")
public class Objection {

	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
	public Integer id;
	private String hospitalId;
	private String content;
	private String idEncoded;
	
	public Objection()
	{
		
	}
	
	public String getHospitalId() {
		return hospitalId;
	}
	public void setHospitalId(String hospitalId) {
		this.hospitalId = hospitalId;
	}
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}

	public String getIdEncoded() {
		return idEncoded;
	}

	public void setIdEncoded(String idEncoded) {
		this.idEncoded = idEncoded;
	}

	public Integer getId() {
		return id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	
}

File: PharmacyApplication/Pharmacy/Model/Objection.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Pharmacy.Model
{
    public class Objection
    {
        public int Id { get; set; }
        public string HospitalId { get; set; }
        public string Content { get; set; }
        public string IdEncoded { get; set; }

        public Objection() { }

        public Objection(int id, string content, string idEncoded)
        {
            Id = id;
            Content = content;
            IdEncoded = idEncoded;
        }
    }
}
